### DISCUSSION TITLE
[@types/node] Export the MockTracker class type from node:test

### DISCUSSION THREAD

**Discussion by darcyrush** [DefinitelyTyped/DefinitelyTyped#70175] [Issues with a @types package] [answered]
The new `node:test` module allows tests to expose their own `mock` export which is a `MockTracker` instance

```ts
import { describe, it } from "node:test";

const wrap = (mock: any) => {
  // Do stuff
}

describe("Example" () => {
  it("Example", ({ mock }) => {
    // mock is of type MockTracker
    wrap(mock)
  })
})
```
It would be nice to be able to import this type into our tests, as I have come across libraries which allow the passing of a mocking class, and currently I have to use `any` or `unknown` when passing it as an argument to my own wrapper function.

```ts
import type { MockTracker } from "node:test"

const wrap = (mock: MockTracker) => {
  // Do stuff
}
```

If there is a way to do this already, it would be appreciated to have an example.

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
Will be added in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70326

**Reply by darcyrush** [reply]
Looks like this never got merged

**Comment by Renegade334** [comment, marked as answer]
Resolved in #73002.
